services/terraformer/server/domain/utils/backend/backend.go

```diff
@@ -9,11 +9,15 @@ import (
 )
 
 var (
-	minioURL    = envs.MinioURL
-	accessKey   = envs.MinioAccessKey
-	secretKey   = envs.MinioSecretKey
-	dynamoURL   = envs.DynamoURL
-	dynamoTable = envs.DynamoTable
+	minioURL           = envs.MinioURL
+	minioAccessKey     = envs.MinioAccessKey
+	minioSecretKey     = envs.MinioSecretKey
```

Comment:
Would it be possible to just have a `bucket_access_key` and `bucket_secret_key` instead of having minio and aws ? They have both compatible API AFAIK.

Reply:
I can try to use s3_adapter SDK to communicate with minio and try to unify it.